gateway/exporters/statsd/statsd_test.go

```diff
@@ -64,86 +52,179 @@ func TestStatsdExporter_Name(t *testing.T) {
 
 func TestStatsdExporter_Export(t *testing.T) {
 
-	config := &configuration.GatewayConfig{
-		Exporters: &configuration.ExportersConfig{
-			StatsdHost: serverAddress,
-		},
+	config.ZookeeperHosts = []string{os.Getenv("ZOOKEEPER_HOSTS")}
+	connMgr, err := createZKTargets(t)
+	assert.Nil(t, err)
+
+	e := &StatsdExporter{
+		config: config,
 	}
-	config.ExporterMetadataAllowlist = []string{"Account"}
 
-	var connMgr connections.ConnectionManager
+	assert.Nil(t, err)
 
-	zkConn, _, err := zk.Connect(config.ZookeeperHosts, config.ZookeeperTimeout)
+	done := make(chan bool, 1)
+	output := &MutexStringSlice{
+		s: []string{},
+	}
 
-	connMgr, err = connections.NewZookeeperConnectionManagerDefault(config, zkConn, nil)
+	go func() {
+		output.mu.Lock()
```

Comment:
This mutex isn't very useful right now. There's a single writer that adds one item at a time, while the slice reader doesn't use it. You may want to use the mutex when accessing the slice.